import-service/lambdas/importFileParser.js

```diff
@@ -0,0 +1,45 @@
+import * as S3 from 'aws-sdk/clients/s3';
+import parse from 'csv-parser';
+
+export const handler = async event => {
+
+  const BUCKET = event.Records[0].s3.bucket.name;
+
+  const s3 = new S3({region: 'eu-central-1'});
+
+  let resault =[], stCode = '';
+  
+  try {
+    for (const record of event.Records) {
+
+      let key = record.s3.object.key,
+          param = { Bucket: BUCKET, Key: key };
+
+      s3.getObject(param).createReadStream()
```

Comment:
Looks like such implementation can cause an issue. It is better to updated implementation by adding async/await or Promise. Does it work properly?

Reply:
Yes. It works as expected.
Here you can find example:
https://stackoverflow.com/questions/55378526/how-does-the-createreadstream-works-in-aws-sdk